lib/internal/errors.js

```diff
@@ -256,7 +257,7 @@ class SystemError extends Error {
 
   [SymbolFor('nodejs.util.inspect.custom')](recurseTimes, ctx) {
     return lazyInternalUtilInspect().inspect(this, {
-      ...ctx,
+      ...new SafeArrayIterator(ctx),
```

Comment:
`ctx` is not an array.

Reply:
This is [object spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals), not iterable spread.
```suggestion
      ...ctx,
```